src/purva/ScannerClass/StringArrayAssignment_16/StringArray.java

```diff
@@ -0,0 +1,34 @@
+package purva.ScannerClass.StringArrayAssignment_16;
+
+import java.util.Scanner;
+
+ public class StringArray {
+		int count ;
+		char character;
```

Comment:
You don't require to create this at instance level you can use this as local variable (I mean you should define this as local variable as this is useful to only single method).